cli/command/image/push.go

```diff
@@ -66,5 +67,8 @@ func RunPush(dockerCli command.Cli, opts pushOptions) error {
 	}
 
 	defer responseBody.Close()
-	return jsonmessage.DisplayJSONMessagesToStream(responseBody, dockerCli.Out(), nil)
+	if !opts.quiet {
+		return jsonmessage.DisplayJSONMessagesToStream(responseBody, dockerCli.Out(), nil)
```

Comment:
Instead of returning nothing, we should make the output useful, so that it can be used when chaining commands (e.g. `IMAGE=$(docker image push -q foobar/foo:1.1.1); docker service create --name foo $IMAGE`).

See my comment here; https://github.com/moby/moby/issues/13588#issuecomment-283489267, and this PR; https://github.com/docker/cli/pull/882, which is implementing this for `docker image pull`

Reply:
--quiet now prints the fqn for the image in the repository